README.md

```diff
@@ -26,29 +33,8 @@ bundle
 rackup
 ```
 
-## Philosophy
-Praxis is a practical implementation of a few guiding principles. In part:
-
-### REST APIs should be consistent
-REST APIs should follow consistent design in patterns and action semantics.
-This includes software patterns for creating applications like code
-organization; application patterns like logging, middleware, query filtering,
-and application bootstrapping; and API routing structures and REST verb
-semantics for exposed resources.
+Or better yet, checkout a full [blog example app](https://github.com/rightscale/praxis-example-app) which showcases a few of the main design and implementation aspects that Praxis has to offer.
```

Comment:
Not entirely a full app yet. It's more of a showcase/tutorial aid at this point than a useful app.